### PR TITLE
Add support for Openoffice Spreadsheet Template (ots)

### PR DESCRIPTION
**Pull request by srimantahyland**
https://alfresco.atlassian.net/browse/HXCS-1839

### CODE REVIEW

**Comment by montgolfiere** [comment]
FYI: for cross-reference, I see `ots` target here:

- https://github.com/jodconverter/jodconverter/blob/master/jodconverter-core/src/main/java/org/jodconverter/core/document/DefaultDocumentFormatRegistry.java#L211
- https://github.com/jodconverter/jodconverter/blob/master/jodconverter-core/src/main/resources/document-formats.json#L191-L202

Note: I could not find (via GitHub search) a specific reference to `sldx ` target.

Background:

- https://github.com/Alfresco/alfresco-jodconverter forked from https://github.com/mirkonasato/jodconverter
- https://github.com/mirkonasato/jodconverter is now archived and refers to other actively maintained repos, eg.
- https://github.com/sbraconnier/jodconverter/ which has since been renamed to https://github.com/jodconverter/jodconverter

**Review by montgolfiere — APPROVED** [review]
Seems ok. I assume this change is as per links in my previous comment.

cc @alandavis @PunchedChimera

**Comment by montgolfiere** [comment]
I've also added a few reviewers (as an fyi) from AlfNux BDU - ie. those who last made commits/releases to/for this project.

File: jodconverter-core/src/main/java/org/artofsolving/jodconverter/document/DefaultDocumentFormatRegistry.java [hunk 1]
```diff
@@ -125,6 +125,11 @@ public DefaultDocumentFormatRegistry() {
         tsv.setStoreProperties(DocumentFamily.SPREADSHEET, tsvLoadAndStoreProperties);
         addFormat(tsv);
 
+        DocumentFormat ots = new DocumentFormat(" OpenDocument Spreadsheet Template", "ots", "application/vnd.oasis.opendocument.spreadsheet-template");
```
Comment (dsibilio) [inline jodconverter-core/src/main/java/org/artofsolving/jodconverter/document/DefaultDocumentFormatRegistry.java]: Thank you @montgolfiere for the fyi 🙏 

I believe there might be an additional unwanted whitespace here?
```suggestion
        DocumentFormat ots = new DocumentFormat("OpenDocument Spreadsheet Template", "ots", "application/vnd.oasis.opendocument.spreadsheet-template");
```

**Review by dsibilio — CHANGES_REQUESTED** [review]

**Review by dsibilio — CHANGES_REQUESTED** [review]
_P.S.: I'm not an expert when it comes down to this project, but I've noticed that there are no tests in this PR and it might not be too hard to add some? E.g.: look at https://github.com/Alfresco/alfresco-jodconverter/tree/master/jodconverter-core/src/test/resources/documents + https://github.com/Alfresco/alfresco-jodconverter/blob/master/jodconverter-core/src/test/java/org/artofsolving/jodconverter/OfficeDocumentConverterFunctionalTest.java#L40, as it seems there might already be the necessary structure in place to easily add one more format to the list of formats to be tested._

**Review by alandavis — APPROVED** [review]

**Review by dsibilio — APPROVED** [review]

**Review by damianujma — APPROVED** [review]

File: jodconverter-core/src/main/java/org/artofsolving/jodconverter/document/DefaultDocumentFormatRegistry.java (hunk 1 above)
Comment (montgolfiere) [reply jodconverter-core/src/main/java/org/artofsolving/jodconverter/document/DefaultDocumentFormatRegistry.java]: Good spot ! Resolved by @srimantahyland

**Pull request merged**